controllers/util/provider.go

```diff
@@ -41,15 +43,16 @@ func GenerateClientOnboardingToken(tokenLifetimeInHours int, privateKeyPath stri
 
 // GeneratePeerOnboardingToken generates a ocs-peer token valid for a duration of "tokenLifetimeInHours".
 // The token content is predefined and signed by the private key which'll be read from supplied "privateKeyPath".
-func GeneratePeerOnboardingToken(tokenLifetimeInHours int, privateKeyPath string) (string, error) {
+func GeneratePeerOnboardingToken(tokenLifetimeInHours int, privateKeyPath string, storageClusterUID types.UID) (string, error) {
```

Comment:
Please cap only the first letter of an abbreviation 
```suggestion
func GeneratePeerOnboardingToken(tokenLifetimeInHours int, privateKeyPath string, storageClusterUid types.UID) (string, error) {
```

Reply:
golang-lint is failing with this change

Reply:
I think it is a golang lint issue, @iamniting might be able to give some advice here, or suggest a separate fix for this.
In the meantime, we can merge this (assuming no other issues) without this change.